File: MODUL.1/bahasa.py/PRAK110_2310817120010_NurHikmah.py
import math

C = 5
A = 12
B = int(math.sqrt(A ** 2 + C ** 2))
K = A + B + C
L = int(1 / 2 * C * A)

print(f"Diketahui : \nAlas = {C} cm\nTinggi = {A} cm\n")
print(f"Jawab : \nSisi A = {A} cm\nSisi B = {B} cm\nSisi C = {C} cm")
print(f"Keliling = {K} cm\nLuas = {L} cm²")

File: MODUL.1/bahasa.c/PRAK110_2310817120010_NurHikmah.c
#include <stdio.h>
#include <math.h>
void main()
{
    int A, B, C, pythagoras, keliling, luas;
    C = 5;
    A = 12;
    pythagoras = pow(A, 2) + pow(C, 2);
    B = sqrt(pythagoras);
    keliling = A + B + C;
    luas = (C * A) / 2;
    printf("Diketahui : \nAlas = %d cm \nTinggi = %d cm \n\n", C, A);
    printf("Jawab : \nSisi A = %d cm \nSisi B = %d cm \nSisi C = %d cm \n", A, B, C);
    printf("Keliling = %d cm \nLuas = %d cm persegi", keliling, luas);
}